.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1015,8 +1046,12 @@ stages:
         dockerComposeFileArgs: |
           baseImage=$(baseImage)
           framework=$(publishTargetFramework)
-        dockerComposeCommand: run --no-deps --rm -e baseImage=$(baseImage) -e framework=$(publishTargetFramework) -e USE_NATIVE_LOADER=true -e IncludeTestsRequiringDocker=false IntegrationTests
+        dockerComposeCommand: run --no-deps --rm -e baseImage=$(baseImage) -e framework=$(publishTargetFramework) -e DD_LOGGER_API_KEY=$(DD_LOGGER_API_KEY) -e DD_SERVICE=$(DD_SERVICE) -e DD_ENV=$(DD_ENV) -e TF_BUILD -e BUILD_DEFINITIONNAME -e BUILD_BUILDID -e BUILD_SOURCESDIRECTORY -e BUILD_REPOSITORY_URI -e BUILD_SOURCEBRANCH -e BUILD_SOURCEVERSION -e SYSTEM_TEAMFOUNDATIONSERVERURI -e SYSTEM_TEAMPROJECT -e SYSTEM_TEAMPROJECTID -e SYSTEM_JOBID -e SYSTEM_TASKINSTANCEID -e SYSTEM_PULLREQUEST_SOURCEREPOSITORYURI -e SYSTEM_PULLREQUEST_SOURCEBRANCH -e SYSTEM_PULLREQUEST_SOURCECOMMITID -e USE_NATIVE_LOADER=true -e IncludeTestsRequiringDocker=false IntegrationTests
```

Comment:
Instead of adding all these extra `-e` values, I would add them to the targets in docker-compose.yml instead